taqueria-plugin-archetype/README.md

```diff
@@ -22,18 +21,18 @@ No additional configuration is available
 
 ## Usage
 
-The Archetype plugin exposes a `taq compile` task in Taqueria which can target one, or all the Archetype contracts in the `contracts` folder and compile them to Michelson `.tz` files output to the `artifacts` folder
+The Archetype plugin exposes a `taq compile` task in Taqueria which can target a single Archetype contract in the `contracts` folder and compile it to a Michelson `.tz` file output to the `artifacts` folder
 
-The Archetype plugin also exposes a contract template via the `taq create archetypeContract <contractName>` task. This task will create a new Archetype contract in the `contracts` directory, insert archetype contract boilerplate, and register the contract with Taqueria
+The Archetype plugin also exposes a contract template via the `taq create archetypeContract <contractName>` task. This task will create a new Archetype contract in the `contracts` directory and insert archetype contract boilerplate for you
```

Comment:
Oh! I was not aware of this task, we might want to add it to TVsCE @hu3man

Reply:
Good catch @AlirezaHaghshenas! Yes we should add the template tasks to VScE (Ligo and Archetype)